src/actions/analytics.jsx

```diff
@@ -0,0 +1,90 @@
+
+let Analytics = {
+
+  tagManagerIsLoaded: false,
+
+  EVENT_CATEGORY: 'Consumer Complaint Search',
+
+  /**
+   * Get data layer object.
+   * @param {string} action Name of event.
+   * @param {string} label DOM element label.
+   * @param {string} category Type of event.
+   * @param {Function} callback Function to call on GTM submission.
+   * @param {number} timeout Callback invocation fallback time.
+   * @returns {object} Data layer object.
+   */
+  getDataLayerOptions: function( action, label, category, callback, timeout ) {
+    return {
+      event:         category || Analytics.EVENT_CATEGORY,
+      action:        action,
+      label:         label || '',
+      eventCallback: callback,
+      eventTimeout:  timeout || 500
+    }
+  },
+
+  /**
+   * Initialize the Analytics module.
+   */
+  init: function() {
+    // detect if Google tag manager is loaded
+    if ( window.hasOwnProperty( 'google_tag_manager' ) ) {
+      Analytics.tagManagerIsLoaded = true;
+    } else {
+      var _tagManager;
+      Object.defineProperty( window, 'google_tag_manager', {
+        enumerable: true,
+        configurable: true,
+        get: function() {
+          return _tagManager
+        },
+        set: function( value ) {
+          _tagManager = value;
+          Analytics.tagManagerIsLoaded = true;
+        }
+      } )
+    }
+  },
+
+  /**
+   * @name sendEvent
+   * @kind function
+   *
+   * @description
+   * Pushes an event to the GTM dataLayer.
+   * @param {object} dataLayerOptions Type of event.
+   */
+  sendEvent: function( dataLayerOptions ) {
+    console.log('SEND EVENT GA FUNCTION FIRED')
```

Comment:
Can we get rid of the console line?

Reply:
Other linting:

```
   1:1   warning  Too many blank lines at the beginning of file. Max of 0 allowed  no-multiple-empty-lines
   2:5   warning  'Analytics' is never reassigned. Use 'const' instead             prefer-const
  59:5   warning  Unexpected console statement                                     no-console
  59:16  warning  There must be a space inside this paren                          space-in-parens
  59:47  warning  There must be a space inside this paren                          space-in-parens
```

Reply:
The latest updates should take care of the these linting issues - thanks!